### PR TITLE
update docs to explain how to determine channel IDs

### PR DESCRIPTION
**Pull request by jadrales**
#### Summary

I updated the docs for the command line tools to provide explanation on how channel IDs can be determined. 
#### Ticket Link

#1501

### CODE REVIEW

File: source/manage/command-line-tools.rst [hunk 1]
```diff
@@ -174,8 +174,10 @@ mattermost channel
 .. note::
     **{channel} value**
 
-    For the *add*, *archive*, *delete*, *remove* and *restore* commands, you can specfiy the *{channels}* value by {team}:{channel} using the team and channel URLs, or by using channel IDs. For example, in the following URL the *{channels}* value is *myteam:mychannel*:
-
+    For the *add*, *archive*, *delete*, *remove* and *restore* commands, you can specfiy the *{channels}* value by {team}:{channel} using the team and channel URLs, or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
```
Comment (justinegeffen) [inline source/manage/command-line-tools.rst:177]:
```suggestion
    For the *add*, *archive*, *delete*, *remove*, and *restore* commands, you can specify the *{channels}* value by {team}:{channel} using the team and channel URLs, or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
```

File: source/manage/command-line-tools.rst [hunk 2]
```diff
@@ -329,8 +331,8 @@ mattermost channel move
    This command will be replaced in a future release with the mmctl command `mmctl channel move <https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-channel-move>`__.
 
 Description
-    Move channels to another team. The command validates that all users in the channel belong to the target team. Incoming/Outgoing webhooks are moved along with the channel. Channels can be specified by ``[team]:[channel]`` or by using channel IDs.
-
+    Move channels to another team. The command validates that all users in the channel belong to the target team. Incoming/Outgoing webhooks are moved along with the channel. Channels can be specified by ``[team]:[channel]`` or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
```
Comment (justinegeffen) [inline source/manage/command-line-tools.rst:334]:
```suggestion
    Move channels to another team. The command validates that all users in the channel belong to the target team. Incoming/outgoing webhooks are moved along with the channel. Channels can be specified by ``[team]:[channel]`` or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
```

**Review by justinegeffen — COMMENTED** [review]
Just two comments/suggestions. Otherwise, awesome job and congrats on your first PR! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for this great docs contribution! Next steps include a technical review to ensure accuracy and completeness.

**Comment by jadrales** [comment]
Thank you for reviewing! I've applied the suggested changes.

**Review by jespino — APPROVED** [review]
Looks good to me, from the technical stand point. Thanks @jadrales! :tada:

**Pull request merged**